jest.config.js

```diff
@@ -24,8 +18,16 @@ module.exports = {
 		// TODO: Shared Jest config between Gateway and CDK? Monorepo?
 		'cdk',
 	],
+	transformIgnorePatterns: [
+		// Don't try to transform any node_modules except Preact and @testing-library/preact
+		// Since updating to Jest 30 trying to import these dependencies has resulted in
+		// unsupported ESM getting imported instead of CJS. Possibly because due to
+		// some siginificant changes in how Jest handles ESM under the hood?
+		// See:  https://jestjs.io/docs/upgrading-to-jest30#esm-module-support-and-internal-restructuring
+		'/node_modules/.pnpm/(?!(preact|@testing-library))',
+	],
 	transform: {
-		'^.+\\.(t|j)sx?$': [
+		'^.+\\.(ts|tsx|jsx|mjs|module.js)?$': [
```

Comment:
We now need to transform some of code in the `preact` and `@testing-library/preact` packages which contain `mjs` and `module.js` with unsupported ESM code.

These packages have a CJS version which we were previously using, but for some reason when we updated to Jest 30 it broke our imports and caused them to start using the ESM version instead of the CJS versions. My guess its something to do with [Jests internal changes to how it handles ESM under the hood](https://jestjs.io/docs/upgrading-to-jest30#esm-module-support-and-internal-restructuring) caused it to start importing the wrong modules.